src/options.rs

```diff
@@ -5,4 +5,15 @@ use structopt::StructOpt;
 pub struct Options {
     #[structopt(name = "INPUT", help = "Path to the input file.")]
     pub path: String,
+    #[structopt(flatten)]
+    pub machine_opts: MachineOptions,
+}
+
+#[derive(Debug, StructOpt)]
+pub struct MachineOptions {
+    #[structopt(
+        long = "--overflow-check",
+        help = "Execution panics on integer overflow"
+    )]
```

Comment:
Like in the old version of this PR I would say what is the default value or make it a required option.

Reply:
Oh sure I forgot!